includes/class-image-optimisation.php

```
@@ -108,7 +110,7 @@ public function __construct( $options ) {
 			$this->exclude_convert_imgs    = Util::process_urls( $this->options['image_optimisation']['excludeConvertImages'] ?? array() );
 			$this->preload_front_page_urls = Util::process_urls( $this->options['image_optimisation']['preloadFrontPageImagesUrls'] ?? array() );
 			$this->exclude_post_type_imgs  = Util::process_urls( $this->options['image_optimisation']['excludePostTypeImgUrl'] ?? array() );
-			$this->exclude_sizes           = array_map( 'absint', Util::process_urls( $this->options['image_optimisation']['excludeSize'] ?? array() ) );
+			$this->exclude_sizes           = array_map( 'absint', array_map( 'trim', explode( ',', ( $this->options['image_optimisation']['excludeSize'] ?? '' ) ) ) );
```

PHPStan (2.1.55)

[warning] Parameter `#1` $callback of function array_map expects (callable(string): mixed)|null, 'absint' given.


(argument.type)